lang/german.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+if (!defined('WHMCS')) {
+    die('This file cannot be accessed directly');
+}
+
+$_LANG['p360monitoring_label_plan'] = 'Plan';
+$_LANG['p360monitoring_label_additional_servers'] = 'Zusätzliche Server';
+$_LANG['p360monitoring_label_additional_websites'] = 'Zusätzliche Webseiten';
```

Comment:
It must be "Websites", not "Webseiten".

_Webseite_ is a single page of the entire _Website_.